src/main/java/com/teragrep/cfe_39/Config.java

```diff
@@ -0,0 +1,183 @@
+/*
+   HDFS Data Ingestion for PTH_06 use CFE-39
+   Copyright (C) 2022  Fail-Safe IT Solutions Oy
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+ */
+
+package com.teragrep.cfe_39;
+
+import org.apache.logging.log4j.core.config.ConfigurationSource;
+import org.apache.logging.log4j.core.config.Configurator;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Enumeration;
+import java.util.Properties;
+
+public class Config {
+    // kafka
+    private final String queueTopicPattern;
+    private final Properties kafkaConsumerProperties;
+    private static final Logger LOGGER = LoggerFactory.getLogger(Config.class);
+    private final String hdfsPath;
+    private String hdfsuri;
+    private final String queueDirectory;
+    private final String queueNamePrefix;
+    private final String kerberosHost;
+    private final String kerberosRealm;
+    private final String kerberosPrincipal;
+    private final String hadoopAuthentication;
+    private final String hadoopAuthorization;
+    private final String kerberosKeytabUser;
+    private final String kerberosKeytabPath;
+    private final String kerberosTestMode;
+    private long maximumFileSize;
+    private final int numOfConsumers;
+    private final long pruneOffset;
+
+    // TODO: Set up configuration check for important parameters. Remove old unused parameters.
+
+    Config() throws IOException {
+        Properties properties = new Properties();
+        Path configPath = Paths.get(System.getProperty("cfe_30.config.location", System.getProperty("user.dir") + "/etc/application.properties"));
+        LOGGER.info("Loading application config '" + configPath.toAbsolutePath() + "'");
```

Comment:
should use `LOGGER.info("Loading application config <[{}]>", configPath.toAbsolutePath())` instead. More info in confluence

Reply:
Same for all other logger calls.